test/e2e/support/bootstrap.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env jest */
```

Comment:
I don't think we should need either this or the following line anymore, since these are now incorporated into `e2e/.eslintrc.js`